docker/oso-saml-sso/centos7/start.sh

```diff
@@ -13,6 +13,7 @@ fi
 
 echo user:x:$(id -u):0:USER:/root:/bin/bash >> /etc/passwd
 echo group:x:$(id -G | awk '{print $2}'):user >> /etc/group
+reconf_touchfile="/configdata/reconfigure_successful"
```

Comment:
You have hard coded values in two places.  The /path/file is here and also above in docker/oso-saml-sso/centos7/root/config.yml line #115.

Can this be parameterized across files?